src/sections/CompanyHero/index.js

```diff
@@ -1,21 +1,87 @@
-import React from "react";
+import React, { Component } from "react";
 import Section from "root/components/Section";
 import BackgroundVideoText from "root/components/BackgroundVideoText";
+import BackgroundImageText from "root/components/BackgroundImageText";
+import Typography from "root/components/Typography";
+
+import video from "root/assets/videos/text-video.mp4";
+import textBackground from "root/assets/images/text-img.jpg";
 
 const column = 114;
 const gutter = 28;
 const sixColumns = 6 * column + 5 * gutter;
+const fourColumns = 4 * column + 3 * gutter;
+
+const breakpointMobile = 768;
+const breakpointDesktop = 1268;
+
+export default class CompanyHero extends Component {
+  state = {
+    isJsReady: false,
+  };
+
+  componentDidMount = () => {
+    this.setState({ isJsReady: true });
+    window.addEventListener("resize", this.updateDimensions);
```

Comment:
I think we can extract this `jsReady/resize` logic to a high order component later. Something like `withWindowDimensions`

Reply:
Yes, but in another PR